File: python/datastructure/datastructure/LinkedListTest.py
# filename : LinkedListTest.py 

from LinkedList.LinkedList import SinglyLinkedList, DoublyLinkedList 

if __name__ == "__main__": 
    list = DoublyLinkedList() # singly linked list. 
    list.add(322)
    list.add(3211)
    list.add(8883)
    list._print() # print linked list. 
    list.printList() # print linked list. 
    list.remove(322) # remove linked list. 
    list.printList() # print linked list. 
    print(list.search(3211))
    print(list.search(3223232))

File: cpp/datastructure/datastructure/testing/LinkedList_test.cpp
#include "../LinkedList/LinkedList.h"

int main(int argc, char* argv[]) {

    SinglyLinkedList<int> *list = new SinglyLinkedList<int>(); 
    list->add(32); 
    list->add(321); 
    list->add(11); 
    list->printList();

    return 0; 
}